### ISSUE TITLE
Minimal inline CSS on nodes. Needs to make sure that this plays well with Animations.

### ISSUE THREAD

**Issue by linusha** [LivelyKernel/lively.next#1361]

**Comment by linusha** [comment]
We tried implementing this, but there are a lot of edge cases that require more investigation and handling. On the branch in its current for, animating the borderWidth to 0 for example does not work. Animating the borderRadius to 0 does seem to work at first glance, but the value snaps back once the animation is done.
We postpone this for now. The current progress can be found on  https://github.com/LivelyKernel/lively.next/tree/enhancement/minimal-inline-css.

**Comment by linusha** [comment]
We tried implementing this, but there are a lot of edge cases that require more investigation and handling. On the branch in its current for, animating the borderWidth to 0 for example does not work. Animating the borderRadius to 0 does seem to work at first glance, but the value snaps back once the animation is done.
We postpone this for now. The current progress can be found on  https://github.com/LivelyKernel/lively.next/tree/enhancement/minimal-inline-css.
